### ISSUE TITLE
RegionAccessBuilderTest fails on windows

### ISSUE THREAD

**Issue by FStolte** [eclipse/xtext-core#557]
The test `RegionAccessBuilderTest.testMultiWhitespace2` fails when executed on a windows machine. The source for this error seems to be line endings, similar to issue #460.

The actual output from the assert statement contains the carriage return character twice, while the expected output does not: 
![failuretrace](https://user-images.githubusercontent.com/18054693/33137112-4f6cde6e-cfa7-11e7-8864-58fe26902107.png)

**Comment by cdietrich** [comment]
am not sure if

```
	@Test def void testMultiWhitespace2() {
		'''
			1
			/**/
			foo
		'''.toString.trim.toUnixLineSeparator
```

is an appropriate fix. @meysholdt what do you think?

**Comment by cdietrich** [comment]
fixed in 2.14
